src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,116 @@
+        className={`${styles.footer_block} 2xl:flex 2xl:bg-[#000000] 2xl:w-3/4 2xl:rounded-tl-[50px]`}
+      >
+        <div>
+          <div
+            className={`bg-[#1a1713] 2xl:rounded-[50px] 2xl:rounded-br-[0px] pl-[20px] 2xl:pl-[37px] ${styles.logo_image}`}
+          >
+            <Image
+              src={logo.src}
+              alt="Cap X"
+              width={666}
+              height={279}
+              className={`w-3/4`}
+            />
+            <div className="font-bold text-[15px] sm:text-[22px] lg:text-[28px] 2xl:text-[36px] leading-[44px] text-[#EDAF36] mt-[6px] pl-[10px]">
+              Trading made simpler and smarter
+            </div>
+          </div>
+          <div className="bg-[#1A1713] relative hidden 2xl:block">
+            <div
+              className={`${styles.copywrite_text} text-[18px] pt-[41px] pr-[25px] pb-[29px] pl-[14px] 2xl:bg-[#000000]`}
+            >
+              Copyright &copy; CapX 2024. Made in India
+              <div className={`${styles.pseudo_block} hidden 2xl:block`}>
+                <div
+                  className={`${styles.pseudo_curve} hidden 2xl:block`}
+                ></div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="bg-[#1A1713] 2xl:w-1/4 mt-[11spx] rounded-br-[50px] flex flex-col">
+          <div
+            className={`${styles.join_us_text} pl-[30px] py-[26px] 2xl:bg-[#000000] 2xl:text-center 2xl:pl-[47px] 2xl:pr-[56px]`}
+          >
+            <span>Join CapX on</span>
+            <div className={`${styles.pseudo_block} hidden 2xl:block`}>
+              <div className={`${styles.pseudo_curve} hidden 2xl:block`}></div>
+            </div>
+          </div>
+          <div className="flex items-center 2xl:justify-center h-full pl-[30px] 2xl:pl-[0]">
+            <div className="grid grid-cols-4 gap-[15px] md:gap-[30px] lg:gap-[45px] 2xl:grid-cols-2 2xl:gap-[65px]">
+              <Link
+                href="https://www.youtube.com/channel/UCvFzTxpXy2fWmmP_Ge73Qhg"
+                target="_blank"
+              >
+                <Image
+                  src={youTubeLogo.src}
+                  alt="Discord"
+                  width={72}
+                  height={72}
+                  className="cursor-pointer max-w-[30px] max-h-[30px] md:max-w-[45px] md:max-h-[45px] lg:max-w-[60px] lg:max-h-[60px] 2xl:max-w-[72px] 2xl:max-h-[72px]"
+                />
+              </Link>
+              <Link href="https://x.com/CapX_live" target="_blank">
+                <Image
+                  src={twitterLogo.src}
+                  alt="Twitter"
+                  width={72}
+                  height={72}
+                  className="cursor-pointer max-w-[30px] max-h-[30px] md:max-w-[45px] md:max-h-[45px] lg:max-w-[60px] lg:max-h-[60px] 2xl:max-w-[72px] 2xl:max-h-[72px]"
+                />
+              </Link>
+              <Link href="https://www.instagram.com/capx_live/" target="_blank">
```

Comment:
Next Link ?? we generally use this for linking to some page in our directory, as this provides optimisation by prefetching the page. Do we need it for external links as well ?? We might be overkilling here by using Link